src/main/java/dto/LineDto.java

```diff
@@ -0,0 +1,29 @@
+package dto;
+
+import model.Line;
+import model.Link;
+import model.LinkStatus;
+
+import java.util.List;
+
+public class LineDto {
+    private final List<Boolean> linkExistCollection;
+
+    private LineDto(List<Boolean> linkExistCollection) {
+        this.linkExistCollection = linkExistCollection;
+    }
+
+    public static LineDto from(Line line) {
+        List<Boolean> linkExistCollection = line.getLinks().stream()
+                .map(Link::getLinkstatus)
+                .map(LinkStatus::isPresent)
+                .toList();
```

Comment:
Stream을 적극적으로 활용하셨네요 👍🏻 

```suggestion
        List<Boolean> linkExistCollection = line.getLinks().stream()
                .map(link -> link.getLinkStatus().isPresent())
                .toList();
```

위와 같이 한 번에 처리하는 방법도 있어요 :)

Reply:
조언 감사합니다!

`Stream`의 각 라인은 하나의 동작만 하는 것이 가독성이 좋다고 느껴져 위와 같이 작성했습니다만, 어느 것이 더 보기 좋은지 고민하며 작성했던 것 같습니다.
리뷰어님은 간단한 동작 여럿은 한번에 처리하는걸 선호하시나요?

Reply:
정답은 없지만, 메서드가 어떤 역할을 하는지 네이밍이 명확하다면 한 줄로 표현해도 괜찮다고 생각해요!

Reply:
확실히, 네이밍이 명확하다면 한 줄로 표현하더라도 가독성을 해치지 않는 것 같습니다.
코드를 보며 상황에 맞게 선택하도록 하겠습니다!

네이밍의 중요성을 다시 깨닫고 갑니다. 😄